spec/form_spec.cr

```diff
@@ -363,6 +367,16 @@ describe "LuckyRecord::Form" do
         end
       end
     end
+
```

Comment:
It might be worth adding a spec for creating a new line item with the LineItemForm to make sure the form is hooked up correctly